src/Stripe.net/Entities/Sources/StripeThreeDSecure.cs

```diff
@@ -4,13 +4,124 @@
 
     public class StripeThreeDSecure : StripeEntity
     {
+        /// <summary>
+        /// If `address_line1` was provided, results of the check: `pass`, `fail`, `unavailable`, or `unchecked`.
+        /// </summary>
+        [JsonProperty("address_line1_check")]
+        public string AddressLine1Check { get; set; }
+
+        /// <summary>
+        /// If `address_zip` was provided, results of the check: `pass`, `fail`, `unavailable`, or `unchecked`.
+        /// </summary>
+        [JsonProperty("address_zip_check")]
+        public string AddressZipCheck { get; set; }
+
+        /// <summary>
+        /// True if the cardholder went through the authentication flow and their bank indicated that authentication succeeded.
+        /// </summary>
+        [JsonProperty("authenticated")]
+        public bool Authenticated { get; set; }
+
+        /// <summary>
+        /// Card brand. Can be `American Express`, `Diners Club`, `Discover`, `JCB`, `MasterCard`, `UnionPay`, `Visa`, or `Unknown`.
+        /// </summary>
+        [JsonProperty("brand")]
+        public string Brand { get; set; }
+
+        /// <summary>
+        /// The card used to create this 3DS source.
+        /// </summary>
         [JsonProperty("card")]
         public string CardId { get; set; }
 
+        /// <summary>
+        /// Whether the card was automatically updated at some point by Stripe or not.
+        /// </summary>
+        [JsonProperty("card_automatically_updated")]
+        public bool CardAutomaticallyUpdated { get; set; }
```

Comment:
This is purely internal and should not be here (sorry only seeing it now)

Reply:
I'll remove it, but it is returned by the public API though (cf. the sample JSON in #1274)

Reply:
I know, but I also know it should not and that we flagged it and that it should have been fixed already.